mne/io/eyelink/eyelink.py

```diff
@@ -205,10 +204,10 @@ def __init__(
         self.fname = Path(fname)
         self._sample_lines = None  # sample lines from file
```

Comment:
I missed this on the first read, but all this extra "stuff" is *not* supposed to be part of the `RawEyelink` class. You need to put it all in `_raw_extras[0]`. This can be a dict, too. This is going to be a big change so you don't need to do it here -- we can figure it out later. I'll create a new issue about adding a test for this since it's possible there are other readers that need to be updated, too.

Reply:
Thanks for the tip, I was trying to see if there was a way not to bring all this baggage after reading the file.

Will happily to move it all to `_raw_extras`